manifests/leader-worker-set.clusterserviceversion.yaml

```diff
@@ -0,0 +1,456 @@
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - endpoints
+          - pods
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          - secrets
+          - endpoints
+          - configmaps
+          - pods
+          - services
+          - events
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - replicasets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - watch
+          - list
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - cert-manager.io
+          resources:
+          - certificates
+          - issuers
+          verbs:
+          - create
+          - get
+          - list
+          - patch
+          - update
+        serviceAccountName: openshift-lws-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - leader-worker-set
+  links:
+  - name: Leader Worker Set
+    url: https://lws.sigs.k8s.io/docs/overview/
+  - name: Operand Source Code
+    url: https://github.com/openshift/kubernetes-sigs-lws
+  - name: Operator Source Code
+    url: https://github.com/openshift/lws-operator
+  maintainers:
+    - email: support@redhat.com
+      name: Red Hat
+  maturity: alpha
+  minKubeVersion: 1.31.0
```

Comment:
@atiratree @kannon92 Do you have any reservations for this value?

Reply:
sgtm, 4.17 full support ends next month so I would not go lower with the kube version

Reply:
It seems we could go event lower after today's discussion to 4.17 or 4.16.

LWS seems to support that https://lws.sigs.k8s.io/docs/installation/

JobSet on the other hand seems to support only 3 latest versions https://github.com/kubernetes-sigs/jobset?tab=readme-ov-file#production-readiness-status so I will go with 1.31 there.

Reply:
I will leave the decision up to you.

Reply:
I think, 1.31 (i.e. OCP 4.17) is reasonable. We'll deploy this operator at best case in 4.19 operator hub.

Reply:
ok, but fyi, 4.17 corresponds 1.30 https://github.com/openshift/kubernetes/blob/release-4.17/go.mod